dubbo-spring-boot/dubbo-spring-boot-compatible/autoconfigure/src/main/java/org/apache/dubbo/spring/boot/autoconfigure/DubboConfigurationProperties.java

```diff
@@ -282,12 +282,12 @@ public void setMetadataReports(Map<String, MetadataReportConfig> metadataReports
         this.metadataReports = metadataReports;
     }
 
-    public Map<String, MetricsConfig> getMetricses() {
-        return metricses;
+    public Map<String, MetricsConfig> getMetricsMap() {
+        return metricsMap;
```

Comment:
Should not change now. This is a breaking change and will break users' usage when configure metricsConfig in SpringBoot.